python/ray/scripts/scripts.py

```diff
@@ -701,19 +693,6 @@ def start(node_ip_address, address, port, redis_password, redis_shard_ports,
                 "If --head is not passed in, the --include-dashboard"
                 "flag is not relevant.")
 
-        # Wait for the Redis server to be started. And throw an exception if we
-        # can't connect to it.
-        services.wait_for_redis_to_start(
-            redis_address_ip, redis_address_port, password=redis_password)
```

Comment:
Why remove this?

Reply:
This should be redundant now, with the [retries and waits when reading GCS address](https://github.com/ray-project/ray/blob/93b21926eadf113d3ab6e908916ffaefdef92e46/python/ray/node.py#L345) early in the initialization. I restored this though because it doesn't have to be removed yet.